code/obj/machinery/door/feather.dm

```diff
@@ -9,8 +9,8 @@
 	mat_changedesc = FALSE
 	autoclose = TRUE
 	var/broken = FALSE
-	health = 80
-	health_max = 80
+	health = 200
+	health_max = 200 //half of a standard door
```

Comment:
```suggestion
	health_max = 200 //a third of a standard door
```
airlocks have 600 health

Reply:
Honestly idk if comment is needed at all, or just could say fraction of door health, seems like an implementation detail thing

Reply:
Honestly idk if comment needed at all, or could just say fraction of door health, seems like an implementation detail thing using specific fraction

Reply:
People can hover to see the typical value of door health.

Reply:
I'll just remove the comment